Skip fields without content from result.

HeartoLazor opened the pull request:
Example:
before:
https://i.imgur.com/AhCHR9G.png
after:
https://i.imgur.com/5yqGEEl.png

Commit:
Skip fields without content from result.
Example:
before:
https://i.imgur.com/AhCHR9G.png
after:
https://i.imgur.com/5yqGEEl.png

j4nw commented:
Any interest in getting this merged? Would be great to have without fiddling with the npm package.

stephanemagnenat commented on lib/core/builder.js:
```diff
@@ -86,6 +108,11 @@ async function build (dataFilePath, presskit, {
   return path.relative(buildFolder, htmlFilePath)
 }
 
+//Check if a var exists or it's valid
```
The missing space after // yields a lint error that fails test.

HeartoLazor committed:
Update builder.js

Added missing comment space

stephanemagnenat commented:
Actually, there were 4 errors, now 3 left:
```112:23  error  Missing space before function parentheses
113:24  error  Expected '!==' and instead saw '!='
113:27  error  Strings must use singlequote
```

I'm interested in seeing this PR merged as I have a similar need :-)

HeartoLazor committed:
Update builder.js

Fixed issues:
112:23  error  Missing space before function parentheses
113:24  error  Expected '!==' and instead saw '!='
113:27  error  Strings must use singlequote

stephanemagnenat commented:
@solarsailer any chance to merge this PR?
